exercises/clock/.meta/generator/clock_case.rb

```diff
@@ -9,27 +9,49 @@ def name
   end
 
   def workload
-    property == 'equal' ? compare_clocks : simple_test
+    case property
+    when 'create'
+      simple_test
```

Comment:
I find the when/then format works well in this sort of situation.
`when 'create' then simple_test`

Reply:
You mean for when and the block on the same line?

Reply:
yes.

Reply:
I left the else and error message on two lines. Is that fine?